### ISSUE TITLE
Do not set displaystyle=true universally

### ISSUE THREAD

**Issue by opoudjis** [plurimath/plurimath#168]
Unearthed in https://github.com/metanorma/metanorma-standoc/issues/763

Plurimath conversion to MathML is setting displaystyle=true universally:

https://github.com/plurimath/plurimath/blob/28df9c86359ae5943107997f81b65bbf07b61ae1/lib/plurimath/math/formula.rb#L23

It should not: the consumer should have the option of setting this, because they know whether the math expression occurs as a standalone formula, or in a paragraph. (In the latter case, displaystyle=false is needed.)

I suggest that .to_mathml take a displaystyle value as a parameter, defaulting to true.

**Comment by opoudjis** [comment]
OK, wait up:

I invoke Plurimath in my code as:

* Plurimath::Math.parse(lxm_input, "latex").to_mathml
* Plurimath::Math.parse(math, "mathml").to_mathml
* Plurimath::Math.parse(HTMLEntities.new.decode(expr), "asciimath").to_mathml
* Plurimath::Math.parse(math, "mathml").to_asciimath

You've added the option to Plurimath::Math::Formula

Am I going to see it apply to Plurimath::Math.parse? Or do I need to use Plurimath::Math::Formula.parse ?

**Comment by suleman-uzair** [comment]
> OK, wait up:
> 
> I invoke Plurimath in my code as:
> 
> * Plurimath::Math.parse(lxm_input, "latex").to_mathml
> * Plurimath::Math.parse(math, "mathml").to_mathml
> * Plurimath::Math.parse(HTMLEntities.new.decode(expr), "asciimath").to_mathml
> * Plurimath::Math.parse(math, "mathml").to_asciimath
> 
> You've added the option to Plurimath::Math::Formula
> 
> Am I going to see it apply to Plurimath::Math.parse? Or do I need to use Plurimath::Math::Formula.parse ?

This will be used when you convert a Formula to MathML, let's see the example below.

> Plurimath::Math.parse("sum_3^d", :asciimath).to_mathml

This is the previous method(which can be used right now since we are setting true as a default value to display_style) which generates the following output
```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mstyle displaystyle="true">
    <munderover>
      <mo>&#x2211;</mo>
      <mn>3</mn>
      <mi>d</mi>
    </munderover>
  </mstyle>
</math>
```
but now we can pass the value to **to_mathml** like below
> Plurimath::Math.parse("sum_3^d", :asciimath).to_mathml(display_style: false)

which will generate the following output
 ```
<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mstyle displaystyle="false">
    <munderover>
      <mo>&#x2211;</mo>
      <mn>3</mn>
      <mi>d</mi>
    </munderover>
  </mstyle>
</math>
```

I have also released version 0.3.7.
